src/WTG.BulkAnalysis.Runner/SolutionLocator.cs

```diff
@@ -30,7 +30,7 @@ where solutionPath.StartsWith(pathToBranch, StringComparison.OrdinalIgnoreCase)
 			return solutionPaths.ToImmutableArray();
 		}
 
-		static (IEnumerable<string> solutions, string path) LocateBuildXml(string pathToBranch)

```

Comment:
was this picked up by an analyzer, or was it your own decision?

Reply:
[SA1316](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1316.md) picked it up.

Reply:
Hmmmm, I don't like that one. This actually sparked a fun argument between the C# team and the .NET Runtime team, and I find the C# team argument more compelling for camelCasing them.

Reply:
I'm not particularly concerned one way or another, but I do like consistency.

It would have been nice if they included an 'SAX1316' rule going the other way, like they did for some of the other contentious rules.

Reply:
They made it [configurable](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/Configuration.md#tuple-element-names). We could set that easily here, though it may be more difficult on the internal repos.

Reply:
Apparently the arguments over casing [were resolved](https://github.com/dotnet/runtime/issues/27939#issuecomment-531493543), it's not clear to me from this issue exactly what that resolution is, but apparently [StyleCop believes](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3019) it was in favour of the BCL and PascalCase.